config/webpack.common.js

```diff
@@ -84,7 +84,7 @@ module.exports = (prodMode) =>
                 }
             }),
             new CheckerPlugin(),
-            new ExtractTextPlugin('[name].bundle.css'),
+            new ExtractTextPlugin('[name].[chunkhash].bundle.css'),
```

Comment:
Move this out of common and have separate variants for development and production. `chunkhash` reduces incremental build perf when used in development